### PR TITLE
wishlist page

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

**Review by sanket4120 — APPROVED** [review]
rem or em can be used insted of pixels & use variable names for background & colors. Rest looks good.

**Comment by sanket4120** [comment]
Also try to add preview link or screenshots whenever possible.

File: wishlist-page/wishlist-page.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    
+    .navbar a {
+        font-size: 1.7rem;
+        color:whitesmoke;
+        margin: 1rem; 
+      }
+      .navbar a:hover {
+        color: skyblue;
+      }
+    .href{
```
Comment (ishitakeshawani) [inline wishlist-page/wishlist-page.css:31]: You can give some other names of the class instead of using names of attributes.

**Review by ishitakeshawani — APPROVED** [review]

**Review by Shailseen — APPROVED** [review]
Need some changes for your convenience to reuse some codes and understanding to others. otherwise work you awsm.

**Pull request merged**